hack/config.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+usage() {
+    cat <<EOF >&2
+usage:
+
+$0 get-config resourcegroup storageaccountname jsonpath
```

Comment:
simplify! config.sh can work out the storageaccount name for itself (check the tags).  Have it output the config to stdout so you can do `config.sh get-config RG | jq .config.HtPasswd`.

Reply:
Was doing this as I saw your comments :)